contracts/deploy/mainnet/160_upgrade_morpho_strategies.js

```diff
@@ -132,6 +132,43 @@ module.exports = deploymentWithGovernanceProposal(
           signature: "setHarvesterAddress(address)",
           args: [addresses.multichainBuybackOperator],
         },
+        {
+          // 6. Set the Morpho Gauntlet Prime USDC strategy as the default for USDC
+          contract: cVaultAdmin,
+          signature: "setAssetDefaultStrategy(address,address)",
+          args: [
+            addresses.mainnet.USDC,
+            dMorphoGauntletPrimeUSDCStrategyProxy.address,
+          ],
+        },
+        {
+          // 7. Withdraw all from the Morpho Steakhouse strategy
+          contract: cVaultAdmin,
+          signature: "withdrawAllFromStrategy(address)",
+          args: [dMorphoSteakhouseUSDCStrategyProxy.address],
+        },
```

Comment:
This is redundant since `removeStrategy` internally calls `withdrawAll`

Reply:
I see. Thanks